panther_analysis_tool/util.py

```diff
@@ -50,11 +52,20 @@ def id_to_path(directory: str, object_id: str) -> str:
     return path
 
 
-def is_latest() -> bool:
+def get_latest_version() -> str:
     try:
         response = requests.get(f"https://pypi.org/pypi/{PACKAGE_NAME}/json")
-        latest_version = response.json().get("info", {}).get("version", "")
         if response.status_code == 200:
+            return response.json().get("info", {}).get("version", UNKNOWN_VERSION)
+    except Exception:  # pylint: disable=broad-except
+        logging.debug("Unable to determine latest version", exc_info=True)
+    return UNKNOWN_VERSION
+
+
+def is_latest() -> bool:
```

Comment:
Should this method return the bool and the latest version so that you don't have to call get_latest_version() again later? I could see this going either way.